packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -663,7 +663,7 @@ function scheduleCallbackIfUnblocked(
       // Schedule a callback to attempt replaying as many events as are
       // now unblocked. This first might not actually be unblocked yet.
       // We could check it early to avoid scheduling an unnecessary callback.
-      scheduleCallback(NormalPriority, replayUnblockedEvents);
+      scheduleCallback(UserBlockingPriority, replayUnblockedEvents);
```

Comment:
What's this change? It doesn't seem right. The idea is to be able retry as much as possible and there can be other user blocking events that unblocks that. Retrying too much can be somewhat expensive.

Reply:
I wrote this months ago so might be misremembering the details slightly, but it's so the events get put into the queue ahead of the rendering tasks. It's not related setting the priority context. I can take another look at it, I think when we chatted in person about this a while ago you suggested that it's fine that it doesn't interrupt the current render, similar to  upgrading during hydration. Or something like that.

Reply:
Ended up reverting this. Wish I could remember the exact reason I thought it was necessary and why it's not anymore. Most likely I was just mistaken.